src/fu-util-common.c

```diff
@@ -1090,6 +1090,10 @@ fu_util_device_flag_to_string (guint64 device_flag)
 		/* skip */
 		return NULL;
 	}
+	if (device_flag == FWUPD_DEVICE_FLAG_PROXY_UPDATABLE) {
+		/* TRANSLATORS: Device is updatable but via another deviced */
+		return _("Updatable (proxy)");
```

Comment:
I don't think end users need to know that it's a proxy update, to most users it should be the same either way.

Reply:
Good point, will fix.